src/components/calendar/Calendar.tsx

```diff
@@ -15,7 +15,7 @@ export const CalendarSelectionRuleType = {
   all: 'ALL',
   lower: 'LOWER',
   upper: 'UPPER',
-  range: 'RANGE'
+  range: 'RANGE',
```

Comment:
It is the codestyle we use. It is based on that https://github.com/airbnb/javascript#commas